Formula/snort.rb

```diff
@@ -39,6 +39,7 @@ def install
       --enable-reload
       --enable-sourcefire
       --enable-targetbased
+      --disable-control-socket
```

Comment:
Can you please elaborate on why this config option was added?

Reply:
Because the control socket option is only relevant for Linux systems, not for Macs.